tests/class-wp-test-rest-controller-testcase.php

```diff
@@ -39,4 +39,8 @@ public function tearDown() {
 
 	abstract public function test_get_item_schema();
 
+	public function test_rest_url_for_leading_slash( $url, $path ) {
+		// Make sure path for rest_url has a leading slash for proper resolution.
+		$this->assertTrue( 0 === strpos( $path, '/' ) );
```

Comment:
Thanks, that solved everything.  One last thing though, the test itself doesn't really help at all for what we are trying to do as `get_rest_url()` always adds the leading slash before it hits the filter `rest_url`.  How would we test the input into the filter to see if there is a slash or not?  Currently the test itself just checks to make sure there is a leading slash at the start of $path and asserts whether that is true which it is regardless of whether there is a leading slash or not being put into the `rest_url()`.